application/src/main/kotlin/no/kartverket/matrikkel/bygning/application/models/kodelister/EgenregistreringKoder.kt

```diff
@@ -63,3 +63,10 @@ enum class KildematerialeKode(
     Byggesaksdokumenter("Byggesaksdokumenter", "Du har funnet dette i dokumenter laget i forbindelse med en byggesak, for eksempel tegninger, tillatelser eller ferdigattest."),
     AnnenDokumentasjon("Annen dokumentasjon", "Hvis du har noe dokumentasjon som ikke passer i de andre alternativene, men det er mulig å dokumentere valget ditt.")
 }
+
+enum class ProsessKode(
+    override val presentasjonsnavn: String,
+    override val beskrivelse: String
+) : IKode {
+    Egenregistrering("Egenregistrering", "Føringen er en egenregistering gjort av bruker"),
```

Comment:
```suggestion
    Egenregistrering("Egenregistrering", "Føringen er en egenregistrering gjort av bruker ved hjelp av Min Eiendom"),
```

Fikse skriveleif, tenker det er lurt å spesifisere at det er Min Eiendom og. Vil tro at ting kan skille seg der